### DISCUSSION TITLE
ScriptReference/Tilemaps.Tilemap-tilemapTileChanged

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#650] [Notes]
# ScriptReference/Tilemaps.Tilemap-tilemapTileChanged

https://docs.unity3d.com/ScriptReference/Tilemaps.Tilemap-tilemapTileChanged.html

**Comment by andreasIsCoding** [comment]
Delegate type for this event is `System.Action<Tilemap, Tilemap.SyncTile[]>`
